test/integration/common/TypeDBRunners.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.studio.test.integration.common
+
+import com.vaticle.typedb.common.test.TypeDBRunner
+import com.vaticle.typedb.common.test.cluster.TypeDBClusterRunner
+import com.vaticle.typedb.common.test.core.TypeDBCoreRunner
+import kotlin.io.path.Path
+
+object TypeDBRunners {
+    val CLUSTER_RUNNER_SERVER_COUNT = 3
+    val CLUSTER_RUNNER_DATA_PATH = Path("cluster-runner-data-path")
+
+    fun withTypeDB(runnerType: RunnerType = RunnerType.CORE, testFunction: (TypeDBRunner) -> Unit) {
+        val typeDB = when (runnerType) {
+            RunnerType.CORE -> TypeDBCoreRunner()
+            RunnerType.CLUSTER -> TypeDBClusterRunner.create(CLUSTER_RUNNER_DATA_PATH, CLUSTER_RUNNER_SERVER_COUNT)
```

Comment:
We've added support for TypeDB cluster runners in the use of Studio, as was our plan when we set out to make use of runners instead of typedb-extractor. Now we can pass an argument `runnerType` in to determine which TypeDB to use. We assume the use of Core by default.